.azure-pipelines/templates/test-single-linux.yml

```diff
@@ -33,9 +33,9 @@ jobs:
 
   variables:
     ${{ if not(eq(parameters.run_py2_tests, 'true')) }}:
-      TOX_SKIP_ENV: py27.*  # Don't run py2 tests
+      TOX_SKIP_ENV: py2.*  # Don't run py2 tests
```

Comment:
Can we rename this env var to not mention `tox` since we're using hatch now?

Reply:
let's keep that for a following PR

Reply:
maybe `SKIP_ENV_NAME`

Reply:
Agreed, I'll open another PR to update that if that's OK for you

Reply:
separate PR sounds good, I just brought it up so it doesn't slip through the cracks

Reply:
https://github.com/DataDog/integrations-core/pull/13633